cpp/src/utilities/cython.cu

```diff
@@ -686,6 +690,96 @@ void call_sssp(raft::handle_t const& handle,
   }
 }
 
+// wrapper for shuffling:
+//
+template <typename vertex_t, typename edge_t, typename weight_t>
+major_minor_weights_t<vertex_t, weight_t> call_shuffle(
+  raft::handle_t const& handle,
+  vertex_t* edgelist_major_vertices /* [IN] */,  // make_zip_iterator does not accept const !
```

Comment:
See other comment about zip_iterator and const

Reply:
`sort_and_shuffle_values()` assumes a zip iterator of non-const's. Which explains the compiler error. All other invocations of sort_and_shuffle() use zip iterators made of non const iterators / buffers.

Can we resolve this?

Reply:
See above explanation on non-const pre-condition.

Reply:
Done.